roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -510,3 +510,236 @@ etcd_config_dir: /etc/ssl/etcd
 etcd_cert_dir: "{{ etcd_config_dir }}/ssl"
 
 typha_enabled: false
+
+cri_socket: /var/run/dockershim.sock
+
+download_run_once: false
+
+# kubernetes image repo define
+kube_image_repo: "gcr.io/google-containers"
+
+# Arch of Docker images and needed packages
+image_arch: "{{ host_architecture | default('amd64')}}"
+
+# Versions
+kubeadm_version: "{{ kube_version }}"
+etcd_version: v3.3.10
```

Comment:
Please remove this!!!

kubespray-defaults imports defaults from download role. Many people rely on versions being here in their deployments. It is a huge breaking change and I do not support it. Let's keep versions in download role, please.

Reply:
It makes no sense at all to do this. The download role's function is to download containers and binaries. The versions are references across roles etc. which is the purpose of kubespray-defaults.
I don't see how this is a breaking change at all, because if you override the version variables those are still valid.
Can you elaborate on how this will break things?